Source/WebCore/platform/mediastream/mac/BaseAudioSharedUnit.cpp

```diff
@@ -354,15 +353,25 @@ void BaseAudioSharedUnit::handleNewCurrentMicrophoneDevice(CaptureDevice&& devic
 
 void BaseAudioSharedUnit::voiceActivityDetected()
 {
-    if (m_voiceActivityThrottleTimer.isActive() || !m_voiceActivityCallback)
+    if (m_voiceActivityThrottleTimer->isActive() || !m_voiceActivityCallback)
```

Comment:
```suggestion
    if ((m_voiceActivityThrottleTimer  && m_voiceActivityThrottleTimer->isActive()) || !m_voiceActivityCallback)
```

Reply:
that doesn't seem right ; you use `m_voiceActivityThrottleTimer` before creating it